### ISSUE TITLE
Menu sub-items should be navigable with the up/down arrow keys

### ISSUE THREAD

**Issue by mikerodonnell89** [SAP/fundamental-ngx#267]
#### Is this a bug, enhancement, or feature request?

Enhancement

#### Briefly describe your proposal.

As per: https://www.w3.org/WAI/tutorials/menus/application/

Top level menus should be navigable with the tab key, and sub items should be navigable with the arrow keys.

For our menu component we should be able to navigate inside the component with the arrows.
There should be implemented some methods in this component. to easy communicate from outside. 
For example: focusFirstItem, also some event thrown when element is clicked/focused. It will simplify adding keyboard support to future/existing components.

**Comment by droshev** [comment]
@JKMarkowski will update this ticket reflecting our last discussion
